api/src/services/itis-service.ts

```diff
@@ -75,10 +85,32 @@ export class ItisService {
     return response.data.response.docs;
   }
 
+  /**
+   * Returns the parent hierarchy for multiple TSNs
+   *
+   * @param {number[]} tsnIds
+   * @return {*}  {Promise<TSNWithHierarchy[]>}
+   * @memberof ItisService
+   */
+  async getHierarchyForTSNs(tsnIds: number[]): Promise<TSNWithHierarchy[]> {
+    const url = await this.getItisSolrTsnHierarchyUrl(tsnIds);
+
+    defaultLog.debug({ label: 'getHierarchyForTSNs', message: 'url', url });
+
+    const response = await axios.get(url);
```

Comment:
(This is something missing in all of the itis queries it looks like) 

It might be nice to define the type of the raw axios response, so that when we parse `response.data.response.docs` it doesn't just think the whole thing is `any`

```
await axios.get<SomeResponseType>(url)
```

Reply:
Assuming the base solr response is generic, could even do something with generics (to pass in the type of the `docs` field)